verified-reviews/loaders/productList.ts

```diff
@@ -0,0 +1,56 @@
+import { AppContext } from "../mod.ts";
+import { AggregateRating, Product } from "../../commerce/types.ts";
+import { ExtensionOf } from "../../website/loaders/extension.ts";
+import { Ratings } from "../utils/types.ts";
+import { createClient, getProductId } from "../utils/client.ts";
+// export type Props = ConfigVerifiedReviews;
+
+/**
+ * @title Opiniões verificadas - Ratings for Products[]
+ */
+export default function productList(
+	config: any,
+	req: Request,
+	ctx: AppContext
+): ExtensionOf<Product[] | null> {
+	const client = createClient({ ...ctx });
+
+	return async (products: Product[] | null) => {
+		if (!products) {
+			return null;
+		}
+		if (!client) return products;
+
+		const productsIds = products.map(getProductId);
+		// const productsIds = ["55766", "50015", "70411", "66268", "6563"] // MOCK PRODUCTS WITH RATINGS
```

Comment:
How about create a README.md inside the app to explain these products mock? does that work?

Reply:
These ids are specific to a client's website, so it was very specific for anyone testing the bounty and having access to this website id! As a suggestion from another review to remove these points, I will remove them here too!